docs/docs/services/qix-engine/access-control.md

```diff
@@ -31,16 +50,86 @@ docker run
     -S SystemDenyRulePath=/<container folder>/deny.txt
 ```
 
-## Allow versus Deny
+## User
 
-In addition to using _Allow_ rules to grant access, the Qlik Associative Engine
-also supports _Deny_ rules to deny access.
+All rules are evaluated in the context of a _User_. In the rules syntax, the user is represented by the `user`
+object.
 
-!!! Note
-    _Deny_ rules are always applied first, if none matched
-    it will continue with _Allow_ rules.
+### User Attributes
+
+`user` contains all attributes defined in the
+[JWT header used to authenticate the user](../../tutorials/authorization.md#json-web-token).
+
+##### Examples
+
+```json
+{
+  "sub": "john-doe",
+  "name": "John Doe",
+  "employeeType": "developer",
+  "tags": ["research"],
+  "custom": {
+    "country": "sweden"
+  }
+}
+```
+
+Here, `employeeType`, and `custom.country` become attributes on `user` and can be used in a rule condition like this:
+
+```c
+user.employeeType = "developer" and user.custom.country = "sweden"
+```
+
+With the example JWT above, this rule evaluates to `true`.
+
+### User Built-in Functions
+
+#### `IsAnonymous()`
+
+Boolean function for user conditions that returns `true` only if the user requesting access is logged in as anonymous.
+
+##### Syntax
+
+```c
+user.IsAnonymous()
+```
 
-## Actions
+##### Examples
+
+```c
+// Evaluates to `true` if the user is part of the "Developers"
+// organization, and is not anonymous:
+user.org == "Developers" and !user.IsAnonymous()
+```
+
+## Resources
+
+A _Resource_ is a generic concept can represent documents, or objects within documents when writing rules. Resources
+carries attributes which may also be used in your rule conditions. In the rules syntax the resource being accessed is
+represented by the `resource` object.
+
+### Resource Attributes
+
+#### Common Attributes
+
+There are different resource types but all share some common attributes:
+
+Attribute              | Description
+---------------------- | -----------
+resource.id            | The unique identifier for the resource.
+resource.name          | The name of the resource, if any.
+resource._resourcetype | The type of the resource being accessed. Equals:<br/>- `"App"` if accessing a document<br/>- `"App.Object"` if accessing an object within a document |
+resource._actions      | The action carried out on a resource. |
```

Comment:
Why is it called `_actions` in plural form? Can it be multi-valued?
For example, would `resource._actions = {"read", "update"}` check if it is `read` or `update`? The syntax then is a bit confusing since it appears to be a "set against set" comparison with the `=` operator.